packages/code-du-travail-frontend/src/annuaire/__tests__/adresse.service.test.js

```diff
@@ -0,0 +1,26 @@
+import { searchAddress } from "../adresse.service";
+
+const results = {
```

Comment:
You might want to rename or scope this `results` constant as it has the same name as the `results` L19